### DISCUSSION TITLE
"Busy, please try again" when updating people with names

### DISCUSSION THREAD

**Discussion by DSew20** [photoprism/photoprism#1929] [Q&A]
Hello,

I thought I had this all working well.  My library was scanned and updated.  I updated lots of people with names and all was well.

But, I am now getting messages pop-up saying "Busy, please try again" when I try to update a person picture that does not have a name.

Was working fine, no I can not update any records.  Any ideas how to find the cause of this error?

I have tried restarting mariadb and photoprism dockers, this fixes the issue temporarily.  But, after a few edits / minutes I get the errors again.

**Comment by lastzero** [comment]
We've recently implemented a change that prevents updates while another update is still running. Depending on your hardware and the number of pictures / faces, this could take a few seconds up to a minute or so. Does it work again if you wait a little? Any errors or warnings in the logs?

**Reply by emyarod** [reply]
would it be possible to queue updates so that the user doesn't have to wait for one change to complete before making another change?

it's unclear when the background work is finished and the user can resume making changes

**Reply by lastzero** [reply]
The problem is that users will continue to manually add names without the update while they can and will be automatically assigned. So you have more work and the software has more work if you wait with updating.

**Reply by emyarod** [reply]
can the work be batched and delayed/performed later? I don't know, just trying to see if there are ways to prevent the UI from locking up. not sure how it's handled on Google photos for example

**Reply by lastzero** [reply]
Users should have feedback before continuing to manually assigning faces that were already recognized. What hardware and databases do you have, and how many pictures?

**Reply by emyarod** [reply]
I am running on a dual Xeon E5-2630L v2 server, using MariaDB, and have 60k photos and counting

**Reply by lastzero** [reply]
And for how long do you experience locking? Is this with our preview build? It includes lots of improvements for face recognition.

**Reply by emyarod** [reply]
it's anywhere from a few seconds to a few minutes, but I am using the `latest` docker image not `preview`

**Reply by lastzero** [reply]
Then I recommend trying the preview.

**Reply by emyarod** [reply]
will there be any issues moving back and forth between the stable and the previews image?

**Reply by lastzero** [reply]
It will be the new stable version very soon. Should generally not be a problem anyway.

**Reply by emyarod** [reply]
are there additional steps needed to move to the preview image? just updating my image to pull from `photoprism/photoprism:preview` doesn't seem to work properly

**Reply by lastzero** [reply]
See our documentation. Otherwise, please wait a bit until it is released.

**Reply by emyarod** [reply]
just want to mention that this still remains with the preview. unfortunately the UI is locked any time a change is made, so trying to organize the library becomes an extremely long process. sometimes the UI does not lock up, but any changes are still prevented for some amount of time by the  "Busy, please try again" message

**Comment by DSew20** [comment]
It doesn't seem to save if I leave it.  The blue bar at the top of the screen gets to 90% ish, then stops.

There are some error messages "thumb: thumbnail not cached".  Is this something to explore / look to fix?

**Reply by lastzero** [reply]
Possible, is this our latest release? Could be a storage issue.

**Reply by DSew20** [reply]
Thank you.

I'm on release 220107-f5b7ef83, which I believe is the latest.

Re storage issues, there is plenty of space and the folders are accessible.  What specific issues would you suggest I check?

**Reply by lastzero** [reply]
It would help to see the server logs, ideally in debug mode: https://docs.photoprism.app/getting-started/troubleshooting/docker/#viewing-logs

Errors and warnings will be logged permanently and can be displayed here: https://demo.photoprism.app/library/errors

**Comment by DSew20** [comment]
I just tried tagging a photo with a name and it hung again as before.  This is the server log:

`time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/1d12a88c7d2c64b35a95b18088c394f0091ba8fa/3e871040/tile_500 (304) [71.369µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/e5d4d32c85f351abe4f26f6e7b590bd40816eb7e/3e871040/tile_500 (304) [73.587µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/0378054885232253da8b52e8efb6b3720ac5d18f/3e871040/tile_500 (304) [68.308µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/680b1ab1929e0f10c2f0e9354bb757ffea27637f/3e871040/tile_500 (304) [95.476µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/da7f6b9fbfa25ceb02d8b7533c21bb7518872578/3e871040/tile_500 (304) [62.751µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/e718c1a4a567882c3417863e95deafb8977539e5/3e871040/tile_500 (304) [69.298µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/cead561c5f6db6818b7e1dbb1b2aed7a5ff8e630/3e871040/tile_500 (304) [74.425µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/30db8c3383a0c3567f47d3c24d29e61d0d5edd45/3e871040/tile_500 (304) [83.504µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/930f3659add46c7756f34a5596f063361298cc8d/3e871040/tile_500 (304) [59.178µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/319c638f95752b03a4c6b85f9a8ea922e3432173/3e871040/tile_500 (304) [392.088µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/55791d322d0a6afa5007455eabf7a869fed4aa91/3e871040/tile_500 (304) [43.023µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/c866e6a36c023ecae4e5e74c9e77398411769572/3e871040/tile_500 (304) [51.651µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/72a5340b3b542c081634dc8a23ab1b61db07a7e6/3e871040/tile_500 (304) [68.095µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/b00074f7b53bf13cef4409e4c86a3f54222d97aa/3e871040/tile_500 (304) [55.593µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/aade6cde51529f7f0f0e7e9c2ad4f2fd34843d27/3e871040/tile_500 (304) [50.421µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/d219c8cd69db6994503925d578953acf1c12ecaa/3e871040/tile_500 (304) [46.771µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/d9c8eb0a843566843ea76fadb136a9c8d24afb60/3e871040/tile_500 (304) [64.167µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/898fbb2d2ecfa4de3d0c819174d39b72d7b6524a/3e871040/tile_500 (304) [54.975µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/71b56e7ca302d8e4066727e9bdde440b17b8bd63/3e871040/tile_500 (304) [67.306µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/4c96a31b87d15e6f1413ca44fe38c55660b4bbf3/3e871040/tile_500 (304) [85.187µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/158aa652eab7d9442a5f6ab6e5285e6253c67c8e/3e871040/tile_500 (304) [55.374µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/d6bf6e5d6a9eff4c3e4e18bf363de8a6a8f21297/3e871040/tile_500 (304) [39.82µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/0e69a4486d9561d1183b54011496cd4ef5838c29/3e871040/tile_500 (304) [75.863µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/c28b21a92c454abaa01f8ce657f7fb572fa74f36/3e871040/tile_500 (304) [55.33µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/84c8a1399d30b9a8f99ef1e28a236d21146feb72/3e871040/tile_500 (304) [73.054µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/11e80e198cd5f0f7c80dca0ab4b7c8c430ad887e/3e871040/tile_500 (304) [82.125µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/c97b889f72be252e394f2e3ce72eeee25b7f8db1/3e871040/tile_500 (304) [66.228µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/6cd9dbd8032a84aba22635463e7e941ac07196af/3e871040/tile_500 (304) [75.628µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/27f537aa3dec1e6050c32aeaedbcbc4934ccaed8/3e871040/tile_500 (304) [59.283µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/fb420227f5ce7cc5e9708f7c4608edf8929e7e76/3e871040/tile_500 (304) [74.974µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/acf41417e3a2454646b51b853305dd260f735790/3e871040/tile_500 (304) [67.799µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/58afcd703be047ce46c7f34940e6d58ce8bee139/3e871040/tile_500 (304) [64.315µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/afae2ead3cf30186e156dc91265f7aa29b88b47e/3e871040/tile_500 (304) [550.314µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/8f9503b98a6a8cd9b5ae19ede1ded6707ac79843/3e871040/tile_500 (304) [70.494µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/d2bf76d34ff1c48f843d832c87b30098b7ac4e27/3e871040/tile_500 (304) [203.736µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/f6c3322838e5101a29e49bfea10d8cc98baa2b85/3e871040/tile_500 (304) [207.758µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/a780b87375f5cb6a3513224e8aba33fa1e516d2a/3e871040/tile_500 (304) [95.923µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/91562cd0f2214a1532cbbd238e71ae2888e970a2/3e871040/tile_500 (304) [71.297µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/59fff94f73e0b304aec358710f93f50660acf685/3e871040/tile_500 (304) [62.712µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/240b13f94217019de555c0348e77844cd1fccfbf/3e871040/tile_500 (304) [70.81µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/6307d55934d21f493caa2c9d09d364f6f05b80f8/3e871040/tile_500 (304) [131.766µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/d097cc420630512622706d00e0e8c476627a6974/3e871040/tile_500 (304) [64.505µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/04e51789fdea0b9577c6f4a2934d3fdffea2b9e0/3e871040/tile_500 (304) [75.665µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/48b351bb0594125bc8e83576e05dd93735f9f54f/3e871040/tile_500 (304) [114.463µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/9fc3bb3f93558083370c2dbe12304b30374b0e37/3e871040/tile_500 (304) [62.704µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/0ddd731e441cc9a22fab25c462999799162743ac/3e871040/tile_500 (304) [69.907µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/84b9a4e7fa5fff411f4ba69641f7f11a18cd8c9c/3e871040/tile_500 (304) [74.207µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/d652dbcb5ddad38de6f696e211268b946e4d51fd/3e871040/tile_500 (304) [75.756µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/7253b5d4e6634f1fe5653b850e2ff83c28223bd3/3e871040/tile_500 (304) [39.683µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/9230dd38a6b4fc6b643fbb62c2471182254fd070/3e871040/tile_500 (304) [117.37µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/39858e8add19b8c365df04a68156614275112395/3e871040/tile_500 (304) [65.212µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/b99a0e4d531a490efe4f822c7e972cfa37210a15/3e871040/tile_500 (304) [81.658µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/4ab99891b18965b383de355474793bef074cabbc/3e871040/tile_500 (304) [110.63µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/f5325b7629142f3398fc36877cb5940899d2a8c8/3e871040/tile_500 (304) [43.249µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/d83c5ca993f1feb802c69ce9932c21ce6b23a3f4/3e871040/tile_500 (304) [66.79µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/0f8c93ea40007d9f34424c53dd97a20671d71f59/3e871040/tile_500 (304) [136.779µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/a4aaf07b485b56329782e046e82b42ea12855392/3e871040/tile_500 (304) [102.582µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/4c2b6da0f46947c1f4cf550185082d9c9338812f/3e871040/tile_500 (304) [88.733µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/6cb837e033e411b59ed5ccc04d41c4db5f0e9b72/3e871040/tile_500 (304) [97.091µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/5c0c93378f7eaf75f670a17df3f7e930ac97533d/3e871040/tile_500 (304) [67.03µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/95fdb3c43edaca244bfb3f4371557730461eb245/3e871040/tile_500 (304) [63.534µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/092f3cdca356c0e977a92cf8316da41abd1b9d91/3e871040/tile_500 (304) [44.408µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/59bf07135f9c8e309d8272b34d0e38ec60db608c/3e871040/tile_500 (304) [356.609µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/0daea7253279d3550848dd527b13ccc2f5dfdd22/3e871040/tile_500 (304) [74.551µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/d535794f1a7fe5d9fee8f67a255547938afb166c/3e871040/tile_500 (304) [90.165µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/665545038e23b3d19e38fcb7889ab457fc7b812c/3e871040/tile_500 (304) [66.026µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/5ec058dfac6a8c8341614cb0adb0403c843c9ab9/3e871040/tile_500 (304) [75.96µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/684d93dba3d4424f30ff471583df301a7f4e0809/3e871040/tile_500 (304) [889.632µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/53d71f1c6fa663ac923da67327d3ded6dca23ee3/3e871040/tile_500 (304) [63.707µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/bc2793206832c1344b3ecd234b1b412f49622922/3e871040/tile_500 (304) [78.294µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/48205c2c4b95fca8bee90b892817eb097589cbc7/3e871040/tile_500 (304) [157.122µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/55c6f00008b32291a8972dbca1e1b55e19add86c/3e871040/tile_500 (304) [193.002µs]"
time="2022-01-12T18:44:27Z" level=debug msg="http: GET /api/v1/t/1bc527a7b1e76e44b0dc9515d2d606e55cf0647d/3e871040/tile_500 (304) [477.203µs]"
time="2022-01-12T18:44:34Z" level=debug msg="http: GET /api/v1/folders/originals/2018?files=true&uncached=true&count=999&offset=0&q=&all= (200) [378.1123ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET '/api/v1/folders/originals/2018/05 Glamping?files=true&uncached=true&count=999&offset=0&q=&all=' (200) [372.163052ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/4f7208304b155b332aba3abb9816e15bc846a5e2/3e871040/tile_500 (200) [1.247188ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/fc327ead6cffd3ab880d5c78f714ea97bb118c2e/3e871040/tile_500 (200) [6.934278ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/675da2556f08bce95d253a794f44a753a88d1cc2/3e871040/tile_500 (200) [1.445041ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/8baafe3d533ed84aecd8a2b47521f35a9b0ce8b4/3e871040/tile_500 (200) [8.57932ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/48cd463f1588315526a77bb372b5e441a81f3b85/3e871040/tile_500 (200) [9.319767ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/ea4977870dbd2f722334aed27fc366cd9d9453ec/3e871040/tile_500 (200) [8.98512ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/30185c9d45d3e82f342a213db4a02b7c23f5eecc/3e871040/tile_500 (200) [1.352369ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/615be8b6424315bc9253056eb045eb3fe64b7c1c/3e871040/tile_500 (200) [1.239172ms]"
time="2022-01-12T18:44:43Z" level=debug msg="http: GET /api/v1/t/c13dd88865d0c6794567a4e331355fc11f86e4b7/3e871040/tile_500 (304) [91.889µs]"
time="2022-01-12T18:44:44Z" level=debug msg="cached thumbs:bd5bd21c455b58fa25bba1de6f0d583df756651d:tile_500 [106.398967ms]"
time="2022-01-12T18:44:44Z" level=debug msg="http: GET /api/v1/t/bd5bd21c455b58fa25bba1de6f0d583df756651d/3e871040/tile_500 (200) [106.962551ms]"
time="2022-01-12T18:44:46Z" level=debug msg="http: GET /api/v1/svg/photo (200) [156.374µs]"
time="2022-01-12T18:44:46Z" level=debug msg="http: GET /api/v1/photos/pr5hc0t90ekbptn0 (200) [20.873888ms]"
time="2022-01-12T18:44:48Z" level=debug msg="crop: 48cd463f1588315526a77bb372b5e441a81f3b85_720x720_fit.jpg is too small, upscaling 127px to 320px"
time="2022-01-12T18:44:48Z" level=debug msg="crop: 48cd463f1588315526a77bb372b5e441a81f3b85_720x720_fit.jpg is too small, upscaling 119px to 320px"
time="2022-01-12T18:44:48Z" level=debug msg="saved 48cd463f1588315526a77bb372b5e441a81f3b85_320x320_crop_13c0960b110a.jpg"
time="2022-01-12T18:44:48Z" level=debug msg="http: GET /api/v1/t/48cd463f1588315526a77bb372b5e441a81f3b85-13c0960b110a/3e871040/tile_320 (200) [27.773743ms]"
time="2022-01-12T18:44:48Z" level=debug msg="saved 48cd463f1588315526a77bb372b5e441a81f3b85_320x320_crop_2430cc0a60fa.jpg"
time="2022-01-12T18:44:48Z" level=debug msg="http: GET /api/v1/t/48cd463f1588315526a77bb372b5e441a81f3b85-2430cc0a60fa/3e871040/tile_320 (200) [23.487479ms]"
time="2022-01-12T18:44:53Z" level=debug msg="subject: jr5jf8o3izv2w9fd restored"
time="2022-01-12T18:45:52Z" level=debug msg="http: GET /people (200) [5.563728ms]"
time="2022-01-12T18:45:53Z" level=debug msg="http: GET /api/v1/ws (200) [49m49.880808709s]"
time="2022-01-12T18:45:53Z" level=debug msg="http: GET /api/v1/subjects?count=60&offset=0&q=&hidden=&order=relevance&files=1&type=person (200) [1.71745ms]"
time="2022-01-12T18:45:54Z" level=debug msg="http: GET /static/icons/logo/32.png (200) [99.961µs]"
time="2022-01-12T18:45:59Z" level=debug msg="http: GET /api/v1/ws (200) [5.673110111s]"`

**Comment by lastzero** [comment]
Without reading all these logs: do you have a way how we can reproduce this?

**Reply by emyarod** [reply]
this happens during normal operation for me. tagging faces or fixing incorrectly tagged faces results in long pauses after each action. I'm not sure if this is because of library size (165000+ items and counting) but the overall experience requires waiting for a minute or more between those actions

**Comment by lssong99** [comment]
I have exactly issue like this. Docker Image Build [220901-f493607b0] on Synology DS720+.
In the begining the manual tagging face works okay, it took less than 10 seconds for the blue bar to go to 100% then I could manual tag another face. However it took longer time for the blue bar to go to 100% over several days operation.

Now the blue bar will stop at around 95% (visually judge position) and will not go further. 

I could still exit manual tagging screen, but next manual tagging (after couple minutes) will flash "busy, please try again"

I have around 90K photos indexted with 25 people with tagged faces. Total photo count is around 160K (still indexing).

**Reply by graciousgrey** [reply]
If your instance is still busy with indexing it continuously detects new faces and updates face clusters. 
I would suggest to wait until the indexing is done to see if the issue persists.

**Reply by lssong99** [reply]
My instance has completed indexing of photos, but update to people still get the same Busy, please try later message.... Any idea what is needed to provide or what I could to to help debug this?

**Reply by lastzero** [reply]
I'm sure you checked the logs in debug mode yet!?

**Comment by drummonds** [comment]
I am just reporting the same problem.  The log I have is:
faces: marker mrk7ild2po0gncs2 face Q7M3X3LNJKQ73WVPVTKOLDTJUDHF3QJH has ambiguous subjects jrkfe791x3i0zgk9 <> jrkdfj22yjy3cqm5, subject source manual 
The frustrating thing is that I run this on a Synology docker and when I look at the CPU's or space nothing much is happening.  It if went to 100% CPU while busy I would be happier but all dawdling at 1%.  I am on the preview branch

**Reply by graciousgrey** [reply]
Thank you for your feedback!

You are right that in some cases users experience performance issues, or see warnings.

Our next release will focus on OpenID Connect and 2FA support. After that, we'll be happy to look at how we can improve facial recognition.
